### PR TITLE
7289 refactor e2e sentinel tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

medic/cht-core #7289 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by newtewt — CHANGES_REQUESTED** [review]
Nice work on moving over. We can get in line a bit better with chai's expects instead of to.equal everywhere. As it makes more sense to follow the paradigm's within chai instead of what could essentially be `actualValue === expectedValue`

A few callouts that can be updated.

Use the `to.be.empty` on array an empty array check. 

From
 `expect(doc.tasks.length).to.equal(0);`
To
 `expect(doc.tasks).to.be.empty;`
 
https://www.chaijs.com/api/bdd/#empty

Use `to.have.lengthOf(1)` when checking length
From 
`expect(updated[0].tasks.length).to.equal(1);`
To 
`expect(updated[0].tasks).to.have.lengthOf(1)`

Use `to.be.true` or `to.be.false` for true false checks

From 
 `expect(info.transitions.update_clinics.ok).to.equal(true);`
 To 
 `expect(info.transitions.update_clinics.ok).to.be.true;`

File: tests/integration/hooks.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const request = require('request-promise-native');
 exports.mochaHooks = {
   beforeAll: async ()=> {
     console.log('Starting services......');
-    await utils.prepServices();
+    await utils.prepServices({suite:'web'});
```
[the remark this answers is not in the document]
Comment (ngaruko) [reply tests/integration/hooks.js]: refactored this. It is just a boolean atm and it just load the settings.

File: tests/utils.js [hunk 2]
```diff
@@ -928,7 +931,7 @@ module.exports = {
       timeout = setTimeout(() => {
         tail.unwatch();
         reject({ message: 'Timed out looking for details in log files.' });
-      }, 2000);
+      }, 6000);
```
[the remark this answers is not in the document]
Comment (ngaruko) [reply tests/utils.js:938]: It is quite arbitrary. I did a few increments and settled with 6o sec. Looks stable enough.

File: tests/e2e/sentinel/queue.spec.js [hunk 3]
```diff
@@ -127,9 +114,9 @@ describe('Sentinel queue drain', () => {
...
       .then(tombstones => {
         tombstones.forEach(tombstone => {
-          expect(tombstone.type).toEqual('tombstone');
-          expect(tombstone.tombstone).toBeDefined();
+          expect(tombstone.type).to.equal('tombstone');
+          expect(tombstone.tombstone).to.have.property('type', 'data_record');
         });
       });
-  });
+  }).timeout(300000);
```
[the remark this answers is not in the document]
Comment (ngaruko) [reply tests/e2e/sentinel/queue.spec.js]: removed that and it passed build. Will monitor

**Review by newtewt — APPROVED** [review]
Nice job moving these over.

**Pull request merged**